### PR TITLE
Indent HTML and remove spaces from some tags

### PR DESCRIPTION
**Pull request by clemente**
This doesn't change the content, only the formatting:
- it removes `<span>` which add nothing
- it writes some inline elements like `<a>` in a single line instead of many. Also `<em>` and `<strong>`
- it indents everything with 2 spaces (and 4, 6, 8…). No TABs

### CODE REVIEW

**Comment by UmanShahzad** [comment]
@clemente I will have to review this on Sunday, but if that will delay your work, I can just skim it to make sure nothing too important changed and we can merge it. Let me know.

**Comment by clemente** [comment]
@UmanShahzad Sunday is fine, thanks.

File: html/open_edx_devstack_installation_1.html [hunk 1]
```diff
@@ -1,35 +1,25 @@
...
-  </a>
-  or
-  <a href="https://openedx.atlassian.net/wiki/spaces/OpenOPS/pages/60227787/Running+Devstack" target="_blank">
-   complete explanations</a>)
- </li>
+  <li>
+    <a href="https://github.com/open-craft/documentation/blob/master/docker-devstack" target="_blank">a newer Docker version</a>
+    (the
+    <em>README</em>
+    of its GitHub repo has pretty explicit, simple, well-tested steps on how it's setup)
```
Comment (UmanShahzad) [inline html/open_edx_devstack_installation_1.html]: Do these above 3 lines need to be 3 lines? Looks easier to read as 1.

File: html/opencraft_coding_standards_1.html [hunk 2]
```diff
@@ -1,19 +1,13 @@
...
-  <a href="http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html" target="_blank">
-   edX Python Coding Standards
-  </a>
-  and
-  <a href="http://edx-developer-guide.readthedocs.org/en/latest/testing/code-quality.html" target="_blank">
-   code quality standards
-  </a>
- </li>
+  <li>
+    <a href="https://github.com/open-craft/documentation/blob/onboarding-feedback-10-17/pdfs/CodeReviewGuidelinesandGotchas-2017-25-07.pdf" target="_blank">Code Review Guidelines &amp; Gotchas</a>
```
Comment (UmanShahzad) [inline html/opencraft_coding_standards_1.html]: This link is pointing to an old branch.

File: html/open_edx_documentation_official_edx_documentation_1.html [hunk 3]
```diff
@@ -1,13 +1,7 @@
 <p>
- <span>
   Please review the various parts of the
- </span>
- <a href="https://github.com/open-craft/documentation/blob/onboarding-feedback-10-17/official_documentation.md" target="_blank">
-  Official edX documentation
- </a>
- <span>
+  <a href="https://github.com/open-craft/documentation/blob/onboarding-feedback-10-17/official_documentation.md" target="_blank">Official edX documentation</a>
```
Comment (UmanShahzad) [inline html/open_edx_documentation_official_edx_documentation_1.html]: Link pointing to an old branch.

File: html/time_management_and_invoices_tips_from_the_team_members_3.html [hunk 4]
```diff
@@ -1,34 +1,30 @@
...
+    I get a lot of emails from many different projects that we work on,
+    so I have filters that automatically label (most) emails by client,
+    and then I usually go through my emails client by client.
+    So I'll read all the McKinsey emails, then record a time entry for that,
+    then read all the Management emails, then record a time entry for that, etc.
+    That makes it easier to track my time per epic, and since I do it per-client
+    and not per-epic, the labelling itself is not too much admin overhead.
+  </p>
+  <p>
+    <a href="https://github.com/open-craft/documentation/blob/onboarding-feedback-10-17/images/braden-time-tracking.png" target="_blank"><img src="https://github.com/open-craft/documentation/raw/onboarding-feedback-10-17/images/braden-time-tracking.png" alt="" /></a>
```
Comment (UmanShahzad) [inline html/time_management_and_invoices_tips_from_the_team_members_3.html]: Link pointing to an old branch.

**Comment by clemente** [comment]
@UmanShahzad I made the content changes (links) in the other branch, in https://github.com/open-craft/courses/pull/7 , to leave this PR only for indentation changes.
I'll merge this one later and then rebase the other one on top. Thanks

**Comment by clemente** [comment]
Merged. This is a large PR with separate changes (e.g. span, indent, short tags) so I preferred not to squash commits, to make it easier to review them separately in the future if needed.

**Pull request merged**